### PR TITLE
Typescript support

### PR DESCRIPTION
**Pull request by joseluisq**
This PR solves #14 issue.

Feedback welcome!

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export interface LoggerFunction<PrevState = {}, Action = {}, NextState = {}> {
+  (prevState: PrevState, action: Action, nextState: NextState): void
+}
+
+export interface LoggerOptions {
+  log?: LoggerFunction
+}
+
+export interface App<State, Actions> {
```
Comment (joseluisq) [inline src/index.d.ts:13]: __Note:__ `App` interface will be added to `Hyperapp` typings in future https://github.com/hyperapp/hyperapp/pull/727

**Review by joseluisq — COMMENTED** [review]
__Note:__ `App` interface will be added to `Hyperapp` typings in future https://github.com/hyperapp/hyperapp/pull/727

File: src/index.d.ts (hunk 1 above)
Comment (jorgebucaran) [reply src/index.d.ts:13]: Thanks, @joseluisq! Sorry, for the lack of response. I'm mostly focused on V2 at the moment and in my free time, which means I don't have a lot of time to look at your V1 PR. But feel free to leave comments indicating things such as this one and so on on that PR if need be — I'll read every one of them in due time. 👋

Comment (joseluisq) [reply src/index.d.ts:13]: Sure @jorgebucaran I can imagine.
I will keep tuned for news or reviews if necessary.

**Pull request closed without merging**